library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/MaterialTapTargetPrompt.java

```diff
@@ -1563,6 +1577,8 @@ public MaterialTapTargetPrompt create()
             {
                 mPrompt.mTargetView = mTargetView;
                 mPrompt.mView.mTargetView = mTargetView;
+                mPrompt.mView.mDrawRipple = Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB && mIdleAnimationEnabled;
+                mPrompt.mIdleAnimationEnabled = mIdleAnimationEnabled;
```

Comment:
These 2 additions should be outside the if statement so that when targeting a location on the screen (instead of a view) the idle animation can still be disabled.